packages/devtools_app/lib/src/framework/framework_core.dart

```diff
@@ -72,8 +72,8 @@ class FrameworkCore {
   }
 
   /// Gets a VM Service URI from the querystring (in preference from the 'uri'
-  /// value, but otherwise from 'port').
-  static Uri _getUriFromQuerystring(String location) {
+  /// value, but otherwise from 'port' and 'token').
+  static Uri _getUriFromQueryString(String location) {
```

Comment:
we should add a unit test for this method.

Reply:
Added!